test/lib.bash

```diff
@@ -427,39 +427,38 @@ function run_rolling_upgrade_tests {
 
   logger.info "Running rolling upgrade tests"
 
-  local base serving_image_version eventing_image_version eventing_kafka_broker_image_version image_template common_opts images_file
+  local image_template common_opts images_file
 
   # Specify image mapping for REKT tests
   images_file="$(dirname "$(realpath "${BASH_SOURCE[0]}")")/images-rekt.yaml"
-  serving_image_version=${KNATIVE_SERVING_VERSION#knative-}
-  eventing_image_version="${KNATIVE_EVENTING_VERSION#knative-}"
-  eventing_kafka_broker_image_version="${KNATIVE_EVENTING_KAFKA_BROKER_VERSION#knative-}"
+
+  default_knative_eventing_images
+  default_knative_eventing_kafka_broker_images
+  default_knative_serving_images
 
   # Mapping based on https://github.com/openshift/release/tree/master/core-services/image-mirroring/knative
```

Comment:
This comment is out-of-date now. Otherwise this looks good to me now (as long as we get multiarch images later:))

Reply:
Multi arch builds are already been built :tada:

Reply:
Hmm. Which image are multiarch? trying e.g. this and it's "manifest", not "manifest list"
```
ᐅ skopeo inspect --raw docker://quay.io/redhat-user-workloads/ocp-serverless-tenant/serverless-operator-135/kn-serving-autoscaler:latest 
{"schemaVersion":2,"mediaType":"application/vnd.oci.image.manifest.v1+json","config":{"mediaType":"application/vnd.oci.image.config.v1+json","digest":"sha256:90744c7fe55c3b5893b4aff7098dd1392ec60c480a916a37d519ddf8d1cb8e98","size":7755},"layers":[{"mediaType":"application/vnd.oci.image.layer.v1.tar+gzip","digest":"sha256:133388727492505cb9d292d14f00942ad2915ab7df3ec1aabbea88cd25035f7b","size":39460227},{"mediaType":"application/vnd.oci.image.layer.v1.tar+gzip","digest":"sha256:ad929d8e8e974c0686ebc2b5a27fa5ddf62125bc5e19f0cc40a5c1dbd0e9ba47","size":28703160},{"mediaType":"application/vnd.oci.image.layer.v1.tar+gzip","digest":"sha256:7431e963fae10aee616b28966de17f86660d5943dc560bea73e4c077f6997609","size":36262}],"annotations":{"org.opencontainers.image.base.digest":"sha256:d531ff26bffe832391600e8137f01b9d09a712c721ebe966b98d647bd91bc261","org.opencontainers.image.base.name":"registry.access.redhat.com/ubi8/ubi-minimal:latest"}}
```

Reply:
Or you mean, somebody is already working on that but not fully done yet.

Reply:
Serving needs to merge their PR: https://github.com/openshift-knative/serving/pull/912